src/models/emailServiceInt/templates/invitationAdmin.tsx

```diff
@@ -1,23 +1,40 @@
 import Html from '@kitajs/html'
 import { SendMailOptions } from 'nodemailer'
+
 import { Env } from '../../../env/index.js'
+import CompanyHouseEntity from '../../companyHouseEntity.js'
 
 export default {
   name: 'connection_invite_admin' as const,
-  template: async function (env: Env, params: { pin: string; address: string }): Promise<SendMailOptions> {
+  template: async function (
+    env: Env,
+    params: { pin: string; invitator: string; address: string }
+  ): Promise<SendMailOptions> {
+    const companyHouse = new CompanyHouseEntity(env)
+    const localCompany = await companyHouse.localCompanyHouseProfile()
     return {
       to: env.get('EMAIL_ADMIN_ADDRESS'),
       from: env.get('EMAIL_FROM_ADDRESS'),
-      subject: 'Action required: process veritable invitation',
+      subject: `Postal Code for Verification: Invitation from ${params.invitator} on Veritable`,
       text: `
-        Action required: process veritable invitation    
-        PIN: ${params.pin},
-        Address: ${params.address},
+        Hi ${localCompany.company_name},
+        ${params.invitator} as sent you a request to connect securely on Veritable. To complete the verification process, please enter the following 6-digit postal code within your Veritable instance.
+
+        Verification Code: ${params.pin},
+        Address: ${params.address}
       `,
       html: await (
         <>
-          <h1>Action required: process veritable invitation</h1>
-          <p>Pin:</p>
+          <h1>{Html.escapeHtml(`Hi ${localCompany.company_name}`)}</h1>
+          <br />
+          <p>
+            {Html.escapeHtml(
+              `${params.invitator} as sent you a request to connect securely on Veritable. To complete the verification process, please enter the following 6-digit postal code within your Veritable instance.`
+            )}
+          </p>
+
+          <br />
+          <p>Verification Code:</p>
           <p>{Html.escapeHtml(params.pin)}</p>
```

Comment:
This still doesn't match the latest copy in the ticket. Please look again

Reply:
there is a comment that I've used as a refecrence -> https://digicatapult.atlassian.net/browse/VR-263?focusedCommentId=107293

Reply:
It looks like I was using the wrong comment. Confirmed and updated with the correct one.